Lib/test/test_modulefinder.py

```diff
@@ -281,15 +282,6 @@ def _do_test(self, info, report=False, debug=0, replace_paths=[]):
             mf.import_hook(import_this)
             if report:
                 mf.report()
-##                # This wouldn't work in general when executed several times:
-##                opath = sys.path[:]
-##                sys.path = TEST_PATH
-##                try:
-##                    __import__(import_this)
-##                except:
-##                    import traceback; traceback.print_exc()
-##                sys.path = opath
-##                return
```

Comment:
let's leave this here -- don't want to touch unrelated things as part of the patch

Reply:
Okay!

Reply:
I just dropped the commit.